_run/single/kind-config.yaml

```diff
@@ -2,6 +2,7 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
+  image: kindest/node:v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600
```

Comment:
can you use just `v1.19.1`?

Reply:
Following the [kind docs](https://kind.sigs.k8s.io/docs/user/quick-start/#setting-kubernetes-version). They suggest including the sha sum.